installers/Windows/README.md

```diff
@@ -41,4 +41,70 @@ https://github.com/mu-editor/mu/blob/master/win_installer.py
 
 Files covered:
 
-installer.py
\ No newline at end of file
+installer.py
+
+## Step by Step: Creating a Spyder Windows Installer
+To create a Spyder Windows Installer you must follow the following steps.
+
+### Creating a Conda Environment 
+- Download and install Anaconda. You can find the latest version [here](https://www.anaconda.com/products/distribution). 
+- Open the Anaconda prompt and make sure that you have admin permissions.
+- Create the new enviroment with the following commands:
+bash
+$ conda create -n installer python=3.8.10
+$ conda activate installer
+
+The first command creates an enviroment called "installer" with Python version 3.8.10 and the second one, changes the current enviroment to the new one, "installer".'
+
+After changing to the new enviroment, install yarg with pip. 
+bash
+pip install yarg
```

Comment:
Checking the GitHub preview seems like the `bash` command is not recognized so to put text as code you will need to use the triple backtick ` ``` ` as you did for the commands bellow